src/components/Modals/ProjectCardModal/ProjectCardModal.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useEffect, useState } from 'react';
+
+interface PropTypes {
+  open: boolean;
+  onClose: () => void;
+  onViewMoreProjects: () => void;
+}
+
+const ProjectCardModal: React.FC<PropTypes> = ({ open, onClose, onViewMoreProjects }) => {
+  const [answers, setAnswers] = useState<any[]>([
+    { id: 1, answerOne: "Sample answer one", answerTwo: "Sample answer two" }
+  ]);
+  const [answerOne, setAnswerOne] = useState<string>('');
+  const [answerTwo, setAnswerTwo] = useState<string>('');
+  const [isReviewMode, setIsReviewMode] = useState<boolean>(false);
+  const [submissionComplete, setSubmissionComplete] = useState<boolean>(false);
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!answerOne || !answerTwo) {
+      alert("Please answer all the questions.");
+      return;
+    }
+    setIsReviewMode(true);
+  };
+
+  const handleFinalSubmit = () => {
+    const item = {
+      id: Math.floor(Math.random() * 1000),
+      answerOne,
+      answerTwo,
+    };
+    setAnswers((oldList) => {
+      const answers = [...oldList, item];
+      localStorage.setItem("answers", JSON.stringify(answers));
```

Comment:
don't use localStorage like this... reserve local storage for other operations.

Reply:
This is temporary until we connect to the backend and can save the user answers to the database.